client/src/AppRouter.jsx

```diff
@@ -2,23 +2,23 @@ import React from 'react';
 import { Route, BrowserRouter, Switch } from 'react-router-dom';
 import App from './App';
 
-import { Home, Login, Register, Profile, ExerciseEventForm, ExerciseInfo } from './components';
+import { Login, Register, Profile, ExerciseEventForm, ExerciseInfoList, ExerciseEventList } from './components';
```

Comment:
`import` is not officially supported at this time in ES6, the spec may change at any time, please change.